web/src/pages/Route/components/Step1/MetaView.tsx

```diff
@@ -370,7 +385,7 @@ const MetaView: React.FC<RouteModule.Step1PassProps> = ({
       <CustomRedirect />
```

Comment:
Whether `<AppendQueryString />` should be put into this component? `<CustomRedirect />` is the current configuration area for the redirect plugin.

Reply:
This switch can go inside the field group which appears when we do `custom redirection` I suppose.

Reply:
In fact, `append_query_string` and `enable https` can exist at the same time, so it is not suitable to put into `custom redirection`. There is [description](https://apisix.apache.org/docs/apisix/plugins/redirect/#attributes) about `append_query_string`

Reply:
Yes I know, I think you can make it show all the time, as long as the custom redirect is enabled.